addons/website/static/src/systray_items/website_switcher.js

```diff
@@ -36,7 +37,7 @@ WebsiteSwitcherSystray.components = {
 
 export const systrayItem = {
     Component: WebsiteSwitcherSystray,
-    isDisplayed: env => env.services.website.hasMultiWebsites,
+    isDisplayed: env => env.services.website.hasMultiWebsites && env.services.website.websites.length > 1,
```

Comment:
No need of that, keep the behavior as it is right now, the only case where this will help is if someone created a second website (it auto enabled the group, see in website.create) and delete it without creating another one.
Either we judge it is ok (I do, and recent message on the PR seems to be ok too), as it was designed that way, and then we don't need such extra code
Or we don't judge it is ok, and then we need to fix the python unlink python method to remove the group, not doing this js part :grimacing: 

At the end, the website switcher is supposed to be shown when the multi website groupe is enabled, no need for extra check, just need to fix that group behavior if not pleased with